rifle.py

```diff
@@ -181,7 +184,16 @@ def calculate_trajectory(self, initial_position, target_position):
 
         return (x_velocity, y_velocity)
 
+    def check_collisions(self):
+        collisions = self.universe_.get_collisions(self)
+        for collision in collisions:
+            collision.take_damage(1000)
```

Comment:
This line is lightly misleading, that a collision would take damage? I think having `for gameobject in collisions:` would be clearer.